frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/index.tsx

```diff
@@ -73,6 +79,11 @@ function QueryBuilderSearch({
 		return tags;
 	}, [query.aggregateAttribute.key, tags]);
 
+	const isMatricsDataSource = useMemo(
```

Comment:
this does not require useMemo

Reply:
@palashgdev - It is required because `query.datasource` is a mutable prop.